openreview/arr/arr.py

```diff
@@ -387,6 +387,20 @@ def setup(self, program_chair_ids=[], publication_chairs_ids=[]):
             )
 
         setup_arr_invitations(self.invitation_builder)
+
+        self.client.post_group_edit(
+            invitation=self.get_meta_invitation_id(),
+            signatures=[self.venue_id],
+            group=openreview.api.Group(
+                id=self.venue_id,
+                content={
+                    'assignment_quota': {
```

Comment:
can we call it `reviewer_assignment_quota`?

Reply:
I think the value should be added to the request form otherwise it will be removed and added back every time we call to the setup function.